pybv/__init__.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2018 The BIDS Community
+# License: BSD (3-clause)
+"""pybv: a lightweight BrainVision I/O tool"""
+
+__version__ = '0.1'
```

Comment:
I would suggest folowing https://semver.org/ but starting at 0.0.0 for now to make it really obvious that this is an alpha stage.